builder/relay.go

```diff
@@ -193,6 +192,23 @@ func (r *RemoteRelay) SubmitBlockCapella(msg *capella.SubmitBlockRequest, _ Vali
 	return nil
 }
 
+func (r *RemoteRelay) IsPepcRelayer() (bool, error) {
+	log.Info("submitting rob block to remote relay", "endpoint", r.config.Endpoint)
```

Comment:
This log is a little bit misleading, it's not submitting RoB block rather than checking if the relayer is pepc compatible